docs/laravel-integration.md

```diff
@@ -155,8 +158,11 @@ if (SamplingResult::RECORD_AND_SAMPLE === $samplingResult->getDecision()) {
     ->getTracer('io.opentelemetry.contrib.php');
 
     $request = Request::createFromGlobals();
-    $jaegerSpan = $jaegerTracer->startAndActivateSpan($request->getUri());
-    $zipkinSpan = $zipkinTracer->startAndActivateSpan($request->getUri());
+    $jaegerSpan = $jaegerTracer->spanBuilder($request->getUri())->startSpan();
```

Comment:
there is no need to duplicate these (anymore, there may have been in the past) - there should be one tracer with multiple exporters configured

Reply:
Thank you! Addressed